bin/install-sdk

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Error codes + association:
+# 1 - api.github.com request failed
+# 2 - Invalid argument(s)
+# 3 - No $THEOS
+# 4 - Requested SDK DNE
+# 5 - Requested SDK already exists
+# 6 - Failed curl
+
+set -e
+
+urls=$(curl -s https://api.github.com/repos/theos/sdks/releases/latest | grep download_url | sed 's/.*: "\(.*\)"/\1/')
+if [[ -z "$urls" ]]; then
+	echo "ERROR: api.github.com request failed?!" >&2
+	exit 1
+fi
+
+if [[ $# -ne 1 ]]; then
+	echo "Usage: $0 <sdk_name>"
+	echo "Example <sdk_name>s: $(echo "$urls" | sed -e 's/.*\///' -e 's/\.sdk.*//' | tr '\n' ' ')latest latest-tv"
+	exit 2
+fi
+
+if [[ -z "$THEOS" ]]; then
+	echo "ERROR: \$THEOS does not exist." >&2
+	echo "Please install Theos from https://theos.dev/docs/installation before proceeding." >&2
+	exit 3
+elif ! [[ -d "$THEOS/sdks/" ]]; then
+    mkdir -pv "$THEOS/sdks/"
+fi
+
+url=""
+if [[ $1 == latest ]]; then
+    url=$(echo "$urls" | grep 'iPhoneOS' | sort -V | tail -n1)
+elif [[ $1 == latest-tv ]]; then
+    url=$(echo "$urls" | grep 'AppleTVOS' | sort -V | tail -n1)
+else
+    url=$(echo "$urls" | grep "$1")
```

Comment:
this might end up with multiple lines depending what `$1` is

Reply:
`$1` is the full sdk name (e.g., iPhoneOS16.5), so I don't believe that will match more than one result

Reply:
We _expect_ it to be the full SDK name, but only as implied by the sample usage.

`${THEOS}/bin/install-sdk 1` for example errors out.

Reply:
Whoops! Good call. Should be handled now

Reply:
On my machine, the latest check incorrectly passes if `$1` is a substring. 
For example, `bin/install-sdk 11.4` errors out at `curl` because `url` is `AppleTVOS11.4\niPhoneOS11.4`

I like this StackOverflow answer on checking for array membership: https://stackoverflow.com/a/47541882

```sh
printf '%s\0' "${sdks[@]}" | grep -F -x -z -- "${1}"
```

Reply:
Hm, seems like the previous check was for any instance of the str and not an exact match. This works much better as it matches exact strs, thanks!